src/collective/elasticsearch/configure.zcml

```diff
@@ -105,13 +104,24 @@
     />
 
   <!-- CMFCore CatalogTool patches -->
-  <monkey:patch
-    description="unindexObject"
-    class="Products.CMFPlone.CatalogTool.CatalogTool"
-    original="unindexObject"
-    replacement=".patches.unindexObject"
-    preserveOriginal="True"
-    />
+  <configure zcml:condition="have plone-51">
+    <monkey:patch
+      description="unindexObject"
+      class="Products.CMFPlone.CatalogTool.CatalogTool"
+      original="_unindexObject"
+      replacement=".patches.unindexObject"
```

Comment:
Patch on `unindexObject` is now legacy. Please change the method name for `_unindexObject`.